Packages/com.unity.inputsystem/InputSystem/Plugins/UI/UIActionInputModuleEditor.cs

```diff
@@ -0,0 +1,150 @@
+#if UNITY_EDITOR
+
+using UnityEditor;
+using UnityEngine.Experimental.Input.Editor;
+
+namespace UnityEngine.Experimental.Input.Plugins.UI.Editor
+{
+    [CustomEditor(typeof(UIActionInputModule))]
+    internal class UIActionInputModuleEditor: UnityEditor.Editor
```

Comment:
Can we have one of these for `StandaloneInputModule` to have a one-click switch over to this one?